secator/template.py

```
@@ -55,85 +54,192 @@ def print(self):
+				conf = v.copy()
+				opt_name = k
+				conf['prefix'] = f'{node.type} {node.name}'
+				if len(same_opts) > 0:  # opt name conflict, change opt name
+					opt_name = f'{node.name}.{k}'
+					debug(f'[bold]{config.name}[/] -> [bold blue]{node.id}[/] -> [bold green]{k}[/] renamed to [bold green]{opt_name}[/] [dim red](duplicated)[/]', sub=f'cli.{config.name}')  # noqa: E501
+				all_opts[opt_name] = conf
+			return
+
+		# Process task options
+		# a.k.a task options defined in their respective task classes
+		cls = Task.get_task_class(node.name)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)